sklearn/linear_model/_glm/glm.py

```diff
@@ -610,9 +610,15 @@ def _get_loss(self):
 
 
 class GammaRegressor(_GeneralizedLinearRegressor):
-    """Generalized Linear Model with a Gamma distribution.
+    r"""Generalized Linear Model with a Gamma distribution.
 
     This regressor uses the 'log' link function.
+    Its optimization objective is
+
+    .. math::
```

Comment:
In recent days, we tend to not add latex formula since they don't render when looking at the raw docstring.

This info is present to the User Guide which is the location that we favor for such info.